views/style.css

```diff
@@ -11,6 +11,8 @@ slides > slide {
   max-height: 95vh;
   margin-left: calc(95vw / -2);
   margin-top: calc(95vw / (900 / 700) / -2);
+  top: 420px !important;
+  left: calc(400px + 2.5%) !important;
 }
```

Comment:
This replaces a dynamically calculated value with a hardcoded one. Not sure that's universally better. Can you maybe instead improve the formula?

Reply:
Same apply here, I changed the top/left values as 0 to keep margin value minimum.
> That's right. I actually tried to eliminate hardcoded ratio. And with the last commit I shaped it a bit better. So instead of margin calculation i kept the border (margin) fixed to the screen and extract this gap from width and height of the slide `(calc(100% - margin))`. Please check the new commit changes.